docs/mast/mast.rst

```diff
@@ -181,6 +180,111 @@ To get a table of metadata associated with observation or product lists use the
+their descriptions can be found here https://vao.stsci.edu/missionmast/tapservice.aspx/tables#folder38
+
+For a cone search, select_cols would always include ang_sep, sci_data_set_name, search_key and 
+search_position, in addition to the columns specified using select_cols.
+
+for a non cone search, select_cols would always include search_key and sci_data_set_name.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> from astropy.coordinates import SkyCoord
+                >>> missions = MastMissions()
+                >>> regionCoords = SkyCoord(210.80227, 54.34895, unit=('deg', 'deg'))
+                >>> results = missions.query_region(regionCoords, 3, sci_pep_id=12556, select_cols=["sci_stop_time", "sci_targname", "sci_start_time", "sci_status"], sort_by=['sci_targname'])
+                >>> results[:5]
+                <Table masked=True length=5>
+                sci_status   sci_targname   sci_data_set_name       ang_sep        sci_pep_id     search_pos     sci_pi_last_name          search_key        
+                   str6         str16              str9              str20           int64          str18              str6                  str27           
+                ---------- ---------------- ----------------- -------------------- ---------- ------------------ ---------------- ---------------------------
+                    PUBLIC NUCLEUS+HODGE602         OBQU010H0 0.017460048037303017      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010H0
+                    PUBLIC NUCLEUS+HODGE602         OBQU01050 0.017460048037303017      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU01050
+                    PUBLIC NUCLEUS+HODGE602         OBQU01030 0.022143836477276503      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU01030
+                    PUBLIC NUCLEUS+HODGE602         OBQU010F0 0.022143836477276503      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010F0
+                    PUBLIC NUCLEUS+HODGE602         OBQU010J0  0.04381046755938432      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010J0
+
+
+for paging through the results, offset and limit can be used to specify the starting record and the number
+of returned records. the default values for offset and limit is 0 and 5000 respectively.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> from astropy.coordinates import SkyCoord
+                >>> missions = MastMissions()
+                >>> results = missions.query_criteria(sci_start_time=">=2021-01-01 00:00:00", select_cols=["sci_stop_time", "sci_targname", "sci_start_time", "sci_status", "sci_pep_id"], sort_by=['sci_pep_id'], limit=1000, offset=1000)
+                >>> len(results)
+                >>> 1000 
+
+Meta data search can also be performed using object names like M101.
```

Comment:
Suggested rewording:

Metadata queries can also be performed using object names with the `~astroquery.mast.MastMissionsClass.query_object` function.

Reply:
@ceb8 I updated the documentation in the latest commit as per your suggestion.